App.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import {Home, Login} from './src/containers';
+import { StyleSheet, TextInput} from 'react-native';
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
import {StyleSheet, TextInput} from 'react-native';
```